### ISSUE TITLE
Update `sanity/desk` to `sanity/structure`

### ISSUE THREAD

**Issue by mitchuman** [sanity-io/orderable-document-list#77]
If you find a security vulnerability, do NOT open an issue. Email [security@sanity.io](mailto:security@sanity.io) instead.

**Describe the bug**

Update `sanity/desk` to `sanity/structure` as the former is now deprecated.

**To Reproduce**

```ts
import type { StructureBuilder } from 'sanity/desk'
```

**Expected behavior**

```ts
import type { StructureBuilder } from 'sanity/structure'
```

**Screenshots**

**Which versions of Sanity are you using?**

```
@sanity/cli (global)             3.25.0 (up to date)
@sanity/dashboard                 3.1.6 (up to date)
@sanity/eslint-config-studio      3.0.1 (up to date)
@sanity/google-maps-input         3.0.2 (up to date)
@sanity/orderable-document-list   1.1.0 (up to date)
@sanity/vision                   3.25.0 (up to date)
sanity                           3.25.0 (up to date)
```

**What operating system are you using?**

**Which versions of Node.js / npm are you running?**

**Additional context**

**Security issue?**

**Comment by mp-mb** [comment]
Same issue here. This plugin is outdated with the new version and throws TS errors.

**Comment by chasecee** [comment]
Same here, can't use this tool with my old setup (deskTool)

**Comment by Rhym** [comment]
I've had to use https://www.npmjs.com/package/patch-package in the interim.

**Comment by lobolab-website** [comment]
> I've had to use https://www.npmjs.com/package/patch-package in the interim.

Can you show us how you did it?

**Comment by Rhym** [comment]
> > I've had to use https://www.npmjs.com/package/patch-package in the interim.
> 
> Can you show us how you did it?

Sure. pnpm has it built in, but if you're using the package then follow the install instructions and then you can edit the file like so:

```ts
diff --git a/lib/index.d.ts b/lib/index.d.ts
index ad6b188d58d3b8d08c7f8ef2089fcfcebd3bc2a4..c2fe963453d651efd3f15eac3492679209d65dae 100644
--- a/lib/index.d.ts
+++ b/lib/index.d.ts
@@ -1,11 +1,11 @@
 import {ComponentType} from 'react'
 import type {ConfigContext} from 'sanity'
 import {FieldDefinitionBase} from 'sanity'
-import {ListItem} from 'sanity/desk'
-import {MenuItem} from 'sanity/desk'
+import {ListItem} from 'sanity/structure'
+import {MenuItem} from 'sanity/structure'
 import {SortOrdering} from 'sanity'
 import {StringDefinition} from 'sanity'
-import {StructureBuilder} from 'sanity/desk'
+import {StructureBuilder} from 'sanity/structure'
 import {WidenInitialValue} from 'sanity'
 import {WidenValidation} from 'sanity'
```

**Comment by BWeineff** [comment]
Hi all. We recently [released `v1.2.1` of the plugin](https://github.com/sanity-io/orderable-document-list/releases/tag/v1.2.1), which we believe should resolve this issue. Please let us know if the issue persists.
